shared/src/dateHelpers.ts

```diff
@@ -21,12 +21,12 @@ export const getCurrentDateInNewJersey = (): Dayjs => {
   return dayjs().tz("America/New_York");
 };
 
-export const getCurrentDateInNewJerseyISOFormatted = (): string => {
-  return getCurrentDateInNewJersey().toISOString();
+export const getCurrentDateFormatted = (format: string): string => {
+  return getCurrentDate().format(format);
 };
 
-export const getCurrentDateFormatted = (format: string): string => {
-  return dayjs().format(format);
+export const getCurrentDateInNewJerseyFormatted = (format: string): string => {
```

Comment:
[pebble] Should write a test for this function? We have one for `getCurrentDateFormatted`.